src/FunderMaps.Core/Interfaces/Repositories/IAnalysisRepository.cs

```diff
@@ -9,7 +9,9 @@ namespace FunderMaps.Core.Interfaces.Repositories
 {
     public interface IAnalysisRepository
     {
-        Task<AnalysisProduct> GetByIdAsync(Guid userId, string id, CancellationToken token = default);
```

Comment:
This no longer needs the `= default` part if I understand correctly, since we will always get the token from our context now.